doc/json/casebook.json

```diff
@@ -530,7 +530,10 @@
       "timestamp" : "2016-01-01T01:01:01.000Z",
       "revision" : 10,
       "abstraction_level" : "string",
-      "alternate_terms" : [ "string" ],
+      "alternate_terms" : [ {
+        "term" : "string",
```

Comment:
This is acceptable.  We never need more than one "term" per "alternate_terms" value.